src/main/resources/view/PersonListCard.fxml

```diff
@@ -32,9 +32,12 @@
       <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
       <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
       <Label fx:id="subject" styleClass="cell_small_label" text="\$subject" />
+      <Label fx:id="day" styleClass="cell_small_label" text="\$day" />
+      <Label fx:id="begin" styleClass="cell_small_label" text="\$begin" />
+      <Label fx:id="end" styleClass="cell_small_label" text="\$end" />
       <Label fx:id="lesson" styleClass="cell_small_label" text="\$lesson" />
       <Label fx:id="payRate" styleClass="cell_small_label" text="\$payrate" />
-
+      <Label fx:id="paid" styleClass="cell_small_label" text="\$paid" />
```

Comment:
Maybe we could incorporate some additional visual element to paid in future PRs (e.g. icons instead of string values)

Reply:
Noted wtil try to use meaningful text instead